types/powermeter.yaml

```diff
@@ -23,13 +23,56 @@ types:
         type: string
       tariff_id:
         description: Tariff ID
-        type: string
+        type: integer
       cable_id:
         description: Cable ID
-        type: string
+        type: integer
       user_data:
         description: User data
         type: string
+  TransactionRequestStatus:
+    description: Status of a transaction start or stop request.
+    type: string
+    enum:
+        - OK
+        - NOT_SUPPORTED
+        - UNEXPECTED_ERROR
+  TransactionStartResponse:
+    description: Return value when a transaction is started.
+    type: object
+    additionalProperties: false
+    required:
+      - status
+      - transaction_id
+    properties:
+      status:
+        description: Response status that indicates whether the transaction start request could successfully be performed.
+        $ref: /powermeter#/TransactionRequestStatus
+      error:
+        description: If status is not OK, a verbose error message.
+        type: string
+      transaction_min_stop_time:
+        description: Earliest point in time the started transaction can be stopped again (if a minimum duration is required by the meter); yields a RFC3339 timestamp.
```

Comment:
You should add a "format: date-time" to restrict to RFC timestamps

Reply:
added `format: date-time` below